traffic_ops/testing/api/README.md

```diff
@@ -17,86 +17,101 @@
     under the License.
 -->
 
-# Traffic Ops API Tests
+# Traffic Ops Client / API Integration Tests
 
-The Traffic Ops Client API tests are used to validate the clients responses against those from the Traffic Ops API.  
+The Traffic Ops Client API tests are used to validate the clients responses against those from the Traffic Ops API.
 
-The v1 tests are for regression purposes, and the v2 tests were forked from them when Traffic Ops API v2 was merged. All further feature development will only occur in v2.
+## Setup
 
-In order to run the tests you will need the following:
+In order to run the tests you will need a running instance of Traffic Ops and Traffic Ops DB:
 
-1. Port access to both the Postgres port (usually 5432) that your Traffic Ops instance is using as well as the Traffic Ops configured port (usually 443 or 60443).
+1. **Traffic Ops Database** configured port access
+    - _Usually 5432 - should match the value set in database.conf and the **trafficOpsDB port** in traffic-ops-test.conf_
+2. **Traffic Ops** configured port access
+    - _Usually 443 or 60443 - should match the value set in cdn.conf and the **URL** in traffic-ops-test.conf_
+3. Running Postgres instance with a `to_test` database that has empty tables.
```

Comment:
This isn't actually true. The database can have whatever name you want as long as it matches the value set in `database.conf` and `traffic-ops-test.conf`. But also `to_test` is not the default; [the default/sample configuration file in `traffic_ops/testing/api/conf` uses `traffic_ops_development` as the default](https://github.com/apache/trafficcontrol/blob/master/traffic_ops/testing/api/conf/traffic-ops-test.conf#L31).

Reply:
Thats true, however I don't think it can even run successfully using the traffic_ops_development DB, trying right now and its not working

Reply:
https://github.com/apache/trafficcontrol/pull/7108